_posts/2020-07-27-creating-a-snap-chat-mini.markdown

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: "What we Learned Developing a Snapchat Mini (Part 1)"
```

Comment:
@arielj I vote for updating the title to `"The Chronicles of the Minis Master"` :squirrel: